ui/cypress/integration/tests.js

```diff
@@ -8,7 +8,7 @@ describe("Basic tests", () => {
     cy.get("button[id=insert-cohort]").click();
     cy.get("#text").type("New Cohort");
     cy.get("button:Contains(Create)").click();
-    cy.get("button:Contains(Add Criteria)").click();
```

Comment:
Add a brief test for excluded criteria here like the included criteria. Just a couple lines to make sure it's not completely broken.

Reply:
Done.